### PR TITLE
DDP-6179: Added links for different pdf (Spanish/English)

### PR DESCRIPTION
**Pull request by alextanlan**

### CODE REVIEW

File: ddp-workspace/projects/ddp-mbc/src/app/components/data-release/data-release.component.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+                                <span translate>Toolkit.DataRelease.Biases.Text</span>
+                                <a [href]="dataEmailHref" class="Link">{{ infoEmail }}</a>.
+                            </h2>
+                            <ul class="PageContent-ul">
+                                <li class="PageContent-text PageContent-text-list">
+                                    <a href="Methods.pdf" target="_blank" class="Link" translate>Toolkit.DataRelease.Appendices.AppendicesList.Item1.Link</a>
+                                    <span translate>Toolkit.DataRelease.Appendices.AppendicesList.Item1.Text</span>
+                                </li>
+                                <li class="PageContent-text PageContent-text-list">
+                                    <a [href]="currentLanguage === 'es' ? 'AboutYouSurveySpanish.pdf' : 'AboutYouSurveyV2.pdf'"
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-mbc/src/app/components/data-release/data-release.component.html]: Apply different pdf for English/Spanish languages.

No other changes in this file.

**Review by aglushko-broad — DISMISSED** [review]

**Comment by MocanaAtBroad** [comment]
Let's rename those pdfs to make it super-clear what language they are in.
Maybe something like AboutYouSurvey_en.pdf and AboutYouSurvey_es.pdf

**Review by ssettipalli — DISMISSED** [review]
looks good to me

**Comment by alextanlan** [comment]
> Let's rename those pdfs to make it super-clear what language they are in.

Renamed at a1f358d7143e32faf38cfbbda8b8c090a1e9258d

**Review by aglushko-broad — APPROVED** [review]

**Pull request merged**